src/features/bean/Score.js

```diff
@@ -55,9 +55,9 @@ const Field = ({ name, disabled }) => {
 };
 
 /**
- * このコンポーネントは，Scoreページを描写する．
- * @param {bool} disabled trueなら編集可，falseなら不可
- * @returns {Component} Scoreコンポーネント
+ * Clean cup(クリアさ)などの情報を入力するScoreページのコンポーネントを作成する関数
```

Comment:
```suggestion
 * 豆の評価を入力するScoreページの関数コンポーネント
```

あまり具体的に書くとコードを変更したとき面倒なので